web/modules/custom/dpl_go/graphql/dpl_go.base.graphqls

```diff
@@ -1,6 +1,7 @@
 
 type GoLibraryInfo {
   name: String
+  cmsUrl: String
```

Comment:
I think this should be non-nullable. Because the site always knows its url. Right?
Would be nice if have at least some properties that differ from the other optionals. 😅